src/const/navigation.js

```diff
@@ -189,6 +189,9 @@ export const ScreenName = {
   CosmosClaimRewardsValidationError: "CosmosClaimRewardsValidationError",
   CosmosClaimRewardsValidationSuccess: "CosmosClaimRewardsValidationSuccess",
 
+  //crypto_org
+  CryptoEditMemo: "CryptoEditMemo",
```

Comment:
Please rename to `CryptoOrgEditMemo` (in other places too). Thanks.

Reply:
Done 🙏